RedacteurPortaal.Api/Models/Request/UpdateArchiveRequest.cs

```diff
@@ -0,0 +1,23 @@
+﻿using RedacteurPortaal.Api.DTOs;
+using RedacteurPortaal.DomainModels.Media;
+using RedacteurPortaal.DomainModels.NewsItem;
+
+namespace RedacteurPortaal.Api.Models.Request
+{
+    public class UpdateArchiveRequest
+    {
+        public string Title { get; set; }
+
+        public string Label { get; set; }
+
+        public List<MediaPhotoItemDto> MediaPhotoItems { get; set; }
+
+        public List<MediaVideoItemDto> MediaVideoItems { get; set; }
+
+        public List<MediaAudioItemDto> MediaAudioItems { get; set; }
+
+        public List<NewsItemDto> NewsItems { get; set; }
```

Comment:
Willen we op deze manier de archive items aanpassen? Want dan hebben de /VideoItem etc endpoints niet meer nodig